registry/storage/shares.go

```diff
@@ -77,6 +82,7 @@ func (s *sharesStorage) load() error {
 		if err := val.Decode(obj.Value); err != nil {
 			return fmt.Errorf("failed to deserialize share: %w", err)
 		}
+		val.DomainType = types.GetDefaultDomain()
```

Comment:
should probably be passed from `NetworkConfig` to the storage in next version, best to leave a TODO

Reply:
> should probably be passed from `NetworkConfig` to the storage in next version, best to leave a TODO

yeah this will be reverted its just an issue that we have on stage bc we modified domaintype of one of the existing networks which overwrote some shares in the DB :D